examples/README.md

```diff
@@ -0,0 +1,61 @@
+# Examples for Testing Features Locally
+
+The example projects are meant to be used to test features locally by contributors working on this SDK. In the `requirements.txt` file, the deepgram package used is the local version:
```

Comment:
I don't exactly understand the purpose of this `examples` code.

Let's say I'm making a change to the SDK and I'm adding a new feature, `filler_words` to the SDK. I know that I need to add the `filler_words` string to the Options dictionary, so I make those changes.

Now I need to test those changes, so I modify `examples/prerecorded/main.py` to include the `filler_words=true` option. I run `python main.py` and it works!!!

Now I want to commit my additions to the SDK. Do I commit `examples/prerecorded/main.py`?

I'm guessing the answer is "no", and that I should write a unit test instead. If that's the case, I think we want to rename the `examples` directory to something like `testing_examples` to make it clear this code is not meant to contain examples for the user to start their project.

I also think the user _can_ use this code to start their own project. If that's the case, then I think we rewrite this sentence to explain these `prerecorded` and `streaming` projects are basic examples, and they _should_ be used to test contributions to the SDK. -- Maybe this isn't the right thing to do, but I don't understand what exactly this code is supposed to be used for / how it should be used.

Reply:
> Now I want to commit my additions to the SDK. Do I commit examples/prerecorded/main.py?

Correct, the answer is "no". The reality is that sometimes we need to be able to try things out quickly. The example files allow us to make quick changes to the SDKs and test them out within the same repo. We're adding these to all the SDKs.

I don't have a strong opinion about which features are the permanent ones in the `examples/prerecorded/main.py` as the example options to test. But the returned data should be JSON in the terminal so someone can quickly see if the response looks the way it should.

I really hope we can get to a point where people are writing unit tests with every new feature. We're moving in that direction but for now we're still at the stage where we're trying to get more people to actually just test the SDKs thoroughly when people do code reviews.

Reply:
Gotcha, that makes sense now. It might be worth a sentence in the Readme saying that the code changes the person makes to the files in `examples/...` can be discarded when they commit their SDK updates.